Source/Core/Core/HW/HW.cpp

```diff
@@ -70,6 +72,7 @@ void Shutdown()
   DSP::Shutdown();
   AddressSpace::Shutdown();
   Memory::Shutdown();
+  MemoryInterface::Shutdown();
```

Comment:
The order is inconsistent between Init and Shutdown here. Please Shutdown in the reverse order as Init, so put this one between DSP and AddressSpace.